tests/Doctrine/Tests/ORM/Functional/OneToOneEagerLoadingTest.php

```diff
@@ -402,11 +370,8 @@ class TrainOrder
     public $id;
 
     public function __construct(
-        /**
-         * @var Train
-         * @OneToOne(targetEntity="Train", fetch="EAGER")
-         */
-        public $train
```

Comment:
This change is a bit unfortunate. `SlevomatCodingStandard.TypeHints.ParameterTypeHint.MissingNativeTypeHint` is triggered here although it's not just a parameter type that is being added here, but also a property type.

Reply:
Maybe v8.2 does the trick https://github.com/slevomat/coding-standard/releases/tag/8.2.0

Reply:
I'm looking at this again and I don't think I understand what the issue is 🤔.

The `@var` seems redundant so it's added as a native TH.